unit-tests/dds/test-device-init.py

```diff
@@ -64,9 +63,8 @@
             test.check_equal( stream.name(), "s2" )
             test.check_equal( stream.sensor_name(), "sensor2" )
             test.check_equal( stream.type_string(), "motion" )
-            test.check_equal( 1, len( profiles ))
-            # the uid is not communicated any more... therefore 0x1 (the first profile)
-            test.check_equal( '<pyrealdds.motion_stream_profile 0x1 RGB8 30 Hz>', str(profiles[0]) )
+            test.check_equal( len( profiles ), 1 )
+            test.check_equal( str(profiles[0]), '<pyrealdds.motion_stream_profile RGB8 @ 30 Hz>' )
```

Comment:
Its just a communication test but `RGB8` for motion is wierd :-)
Maybe we need to add a motion format specification?

Reply:
There is... but like you said it's not important here.
The next PR will have a whole D435i emulation, with all the profiles including motion.